File: 프로그래머스/0/120815. 피자 나눠 먹기 （2）/피자 나눠 먹기 （2）.java
class Solution {
    public int solution(int n) {
        int answer = 0;
        int lcm = 1; 
        while(true){
            if((lcm % n == 0) && (lcm % 6 == 0)){
            break;
            }
        lcm++;
        }
        return answer = lcm / 6;
    }
}

File: 프로그래머스/0/120815. 피자 나눠 먹기 （2）/피자 나눠 먹기 （2）.js
function solution(n) {
    var answer = 0;
    let lcm = 1; 
    while(true){
      if((lcm % n == 0) && (lcm % 6 == 0)){
        break;
      }
      lcm++;
    }
  	return answer=lcm/6
 
}